scripts/test-check-false-positives.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+cleanup() {
+  r=$?
+
+  # Dump stack logs
+  elastic-package stack dump -v --output "build/elastic-stack-dump/check-${PACKAGE_UNDER_TEST:-${PACKAGE_TEST_TYPE:-httpjson_false_positive_asserts}}"
```

Comment:
Should this have been changed after this comment? https://github.com/elastic/elastic-package/pull/1333/files#r1250740254
```suggestion
  elastic-package stack dump -v --output "build/elastic-stack-dump/check-${PACKAGE_UNDER_TEST:-${PACKAGE_TEST_TYPE:-*}}"
```

Reply:
Instead each package is run in a separate small step in parallel

Reply:
@mrodm Is this the same as we talked or?

Reply:
@jsoriano is right, in this script the default value for `PACKAGE_UNDER_TEST` should be `*` instead of the `httpjson_false_positive_asserts` package. In that regard, this script would be similar to `scripts/test-check-packages.sh`.

Reply:
DOne